### DISCUSSION TITLE
除了Design Token （全局Token, 组件Token）可以配置调节antd的样式，还有其他的方式进行重写吗？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [ant-design/ant-design#44561] [Q&A]
除了Design Token （全局Token, 组件Token）可以配置调节antd的样式，还有其他的方式进行重写吗？

![image](https://github.com/ant-design/ant-design/assets/12727582/bfb53a3d-8ddb-4eb0-b007-5ee02e34f6f8)

比如自己写一个module.css，是否可以对组件进行样式的调节呢？  有现成的案例吗？

**Comment by yakutsuki** [comment]
类名覆盖也是可以的

**Comment by arvinxx** [comment]
在 ConfigProvider 上做全局覆写

**Reply by liaodalin19903** [reply]
您好，请问下，您说的全局覆写，不是指的Design Token对吗？

**Reply by arvinxx** [reply]
对，cp上可以给组件加额外的类名实现覆盖
